doc/tutorials/hires_howto.rst

```diff
@@ -0,0 +1,559 @@
+``keck_hires.flux``, ``keck_hires.coadd1d``, and ``keck_hires.tell``.
+
+The ``keck_hires.flux`` (see also :ref:`flux_file`) looks like this:
+
+.. include:: ../include/keck_hires.flux.rst
+
+It does not need much editing, besides making sure that the ``sensfile`` column has the correct file name.
+In this example, both the quasar and standard star spectra are flux calibrated.
+
+The flux calibration can be run with:
+
+.. code-block:: bash
+
+    pypeit_flux_calib keck_hires.flux
+
+This will add a ``*_FLAM`` columns to each of the spec1d files in the ``Science/`` directory.
+See :ref:`spec1D-datamodel` for more details.
+
+Co-adding
+---------
+
+The next step is to co-add the flux-calibrated spectra. This step is particular important for
+echelle observations, since the orders are are also stitched together. The co-addition is performed
+using the :ref:`pypeit_coadd_1dspec` and the configuration file ``keck_hires.coadd1d``.
+
+The configuration file (see also :ref:`coadd1d_file`) for co-adding the quasar spectra looks like this:
+
+.. include:: ../include/keck_hires.coadd1d.rst
+
+Before running the co-addition, we need to remove the standard star file from the :ref:`data_block`,
+make sure that the ``sensfile`` column has the correct file name, and provide a name for the output ``coaddfile``.
+In this example, the output file is called ``J0100+2802_coadded_50kms.fits``. If the sensitivity function
+used is the same for all the objects, then it is enough to add the file name of the sensitivity function
+only to the first row of the :ref:`data_block`.
+
+In addition, we decided for this example to use a 50 km/s dispersion for the co-added spectrum,
+by setting the parameter ``dv = 50.0``, which allows for a smoother visualization of the final spectrum.
+
+The co-addition can then be run with:
+
+.. code-block:: bash
+
+    pypeit_coadd_1dspec keck_hires.coadd1d
+
+The coadded spectrum can be visualized by just using `matplotlib`_, or we can take advantage of the
+`specutils`_ interface available through PypeIt.  Use of this interface requires to have the `specutils`_
+package installed. See :ref:`spec1D-specutils` for more details.
+This interface allows us to interact with our spectrum using `jdaviz`_.  This **needs** to be done
+within a `jupyter notebook`_, and the following lines can be used to load and visualize the co-added
+spectrum:
+
+.. code-block:: python
+
+    from pypeit.specutils import Spectrum1D
+    from jdaviz import Specviz
+
+    file = 'J0100+2802_coadded_50kms.fits'
+    spec = Spectrum1D.read(file)
+
+    specviz = Specviz()
+    specviz.load_data(spec)
+    specviz.show()
+
+Here is an example of how the spectrum looks in the `jdaviz`_ interface:
+
+.. figure:: ../figures/hires_coadded1d_specviz.png
+
+   The co-added spectrum of the quasar J0100+2802 at z=6.29, with a 50 km/s dispersion, visualized
+   using the `jdaviz`_ interface.
```

Comment:
Nice!  Two things we need to keep an eye on: (1) specutils v2.0 will be released soon and (2) jdaviz can be run as a stand-alone application now.  We'll (likely soon) need to update the docs and the code to accommodate these changes.